### ISSUE TITLE
Disable tag value formatting

### ISSUE THREAD

**Issue by scormaq** [serenity-bdd/serenity-core#1764]
With serenity-cucumber I am able to categorize tests by applying to scenarios my custom tags, like next:
```
  @requirement:SP_5_123 @scope:ShouldBeCamelCase
  Scenario: Login test
...
```
These tags appear next in HTML report. However, Serenity always formats tag values using `Inflector#titleCase` method, which is sometimes undesirable behavior, as tag value loses underscores and camel case.
<details>
<summary>Examples</summary>
<p>

where | current behavior | nice to have
----- | ---------------- | ------------
Main page dashboard | ![formatted_main](https://user-images.githubusercontent.com/6312221/62621164-e9433300-b923-11e9-8b47-450111c9aeee.png) | ![raw_main](https://user-images.githubusercontent.com/6312221/62621180-f2cc9b00-b923-11e9-9b03-1f3eefcbd65e.png)
Scenario page tags | ![tags_formatted](https://user-images.githubusercontent.com/6312221/62621762-6622dc80-b925-11e9-8966-b29740fb6df7.png) | ![tags_raw](https://user-images.githubusercontent.com/6312221/62621787-78047f80-b925-11e9-8988-c374c2191662.png)
Tag page title | ![formatted_title1](https://user-images.githubusercontent.com/6312221/62621851-99656b80-b925-11e9-900a-8c746b3ec125.png) ![formatted_title2](https://user-images.githubusercontent.com/6312221/62621850-99656b80-b925-11e9-924f-5ffc1e53a6d8.png) | ![raw_title2](https://user-images.githubusercontent.com/6312221/62621919-bf8b0b80-b925-11e9-8c8f-651879eaaa30.png) ![raw_title1](https://user-images.githubusercontent.com/6312221/62621922-c023a200-b925-11e9-93e7-e2b4611a9200.png)

</p></details>

Possible solutions:
1. Disable tag value formatting for all tags
2. Introduce new serenity property, say `dashboard.raw.tag.list`, to pass list of tags which should not have formatted values in HTML report.

**Comment by wakaleo** [comment]
Sounds sensible. Could you propose a PR? I think the formatting is in the default.ftl template file (or one of the macro files)

**Comment by scormaq** [comment]
What option is preferable - #1 or #2? I expect that 1st will have simpler fix, but 2nd is more flexible.

**Comment by wakaleo** [comment]
Option 2 would probably be more flexible; a compromise would be to have a property that applies formatting or not to all tags.

**Comment by wakaleo** [comment]
Will be released in version 2.0.70

**Comment by scormaq** [comment]
Works as intended in 2.0.70
